pkg/ddl/reorg.go

```diff
@@ -892,13 +893,13 @@ func (r *reorgInfo) UpdateReorgMeta(startKey kv.Key, pool *sess.Pool) (err error
 	defer pool.Put(sctx)
 
 	se := sess.NewSession(sctx)
-	err = se.Begin()
+	err = se.Begin(context.Background())
```

Comment:
maybe use r.d.ctx

Reply:
that's no much difference with background, it's only cancelled at shutdown. we should refactor to use job worker  ctx or disttask ctx later